### ISSUE TITLE
Add keyboard navigation support for dropdown menus

### ISSUE THREAD

**Issue by Peter112988** [Peter112988/Krishaunina#158]
Dropdown menus are not accessible via keyboard navigation. Implement appropriate key bindings.
